src/tooltip/tooltip.js

```diff
@@ -149,8 +149,10 @@ angular.module('ui.bootstrap.tooltip', ['ui.bootstrap.position', 'ui.bootstrap.s
               if (!tooltip || !tooltip.html()) { return; }
 
               if (!positionTimeout) {
+                tooltip.hide(); // Hide to avoid FOUC until $timeout completes
```

Comment:
`positionTooltip` is called frequently in some cases for an already open tooltip, so this could cause flickering.